### DISCUSSION TITLE
TypeError new viem config

### DISCUSSION THREAD

**Discussion by dylmanning** [wevm/wagmi#2435] [Q&A]
<img width="714" alt="image" src="https://github.com/wagmi-dev/wagmi/assets/5100630/2007e1c0-da20-4079-bf10-14426a3189d5">

May not be a bug, but happy to raise an issue.
But for those of us using strict tsconfigs the default initialisation logic throwing unsafe any call & assignment.

Any ideas?

**Comment by tmm** [comment]
Screenshots are rarely helpful. Can you share a minimal reproduction? TypeScript Playground link is fine.
